docs/system/hardware-metrics.md

```diff
@@ -33,6 +33,18 @@ when creating instruments not explicitly defined in the specification.
 
 <!-- tocstop -->
 
+> **Warning**
+> Existing instrumentations and collector that are using
+> [v1.21.0 of this document](/../v1.21.0/docs/system/hardware-metrics.md)
+> (or prior):
+>
+> * SHOULD NOT adopt any breaking changes from the future versions of this
+>   document system until the system semantic conventions are marked stable.
```

Comment:
```suggestion
>   document until the system semantic conventions are marked stable.
```
I'm not sure what this extra `system` stands for 🤔

Reply:
Good catch. Thanks :)